readthedocs/proxito/middleware.py

```diff
@@ -184,8 +184,19 @@ def add_user_headers(self, request, response):
         with the ``Domain`` object.
         """
         if hasattr(request, 'domain'):
-            domain = request.domain
-            for http_header in domain.http_headers.all():
+            response_headers = [header.lower() for header in response]
+            for http_header in request.domain.http_headers.all():
+                if http_header.name.lower() in response_headers:
+                    log.exception(
```

Comment:
I think we want `error` here, since there isn't an actual exception: https://docs.python.org/3/library/logging.html#logging.Logger.exception
```suggestion
                    log.error(
```

Reply:
You are right!